### ISSUE TITLE
Onboarding (ODS SoC)

### ISSUE THREAD

**Issue by iggisv9t** [newsviz/newsviz#99]
Задача для старта стажировки в рамках ODS SoC:
1. Выбрать открытый датасет для прогона NewsViz
2. Попытаться запустить NewsViz на новом датасете
3. Задокументировать возникшие по пути сложности:
    - Неясные или некорректные инструкции
    - Сложности с установкой
    - Баги
    - Проблемы с данными
    - Любые другие проблемы
4. В результате должны получиться либо Issue с описанием проблем и предполагаемым путём исправления, либо документация (новые доки, исправления или дополнения имеющихся), либо работающий на новых данных инструмент.

Предлагаемые датасеты в списке ниже. Никто их не смотрел внимательно. Они могут не подходить, могут требовать подготовки предварительной. Все неопределённости нужно описывать. Можно выбирать данные из совсем других источников. Например, скачали датасет, открыли, не поняли, как на нём запустить NewsViz: пишите здесь в комментариях что увидели в датасете и вопрос, можно ли как-то его преобразовать для использования инструмента.

Должны быть пригодны:
http://ngodata.ru/dataset/kremlinlaws
http://ngodata.ru/dataset/mosdocs
http://ngodata.ru/dataset/ppgrmeta2020
http://ngodata.ru/dataset/govtwitters

Требуется EDA или выбор источинка:
https://spending.gov.ru/
https://clearspending.ru/
https://openngo.ru/
http://ngodata.ru/dataset
https://hubofdata.ru/dataset

**Comment by pacifikus** [comment]
@iggisv9t  Я правильно понимаю, что по процедуре, которая описана в SoC онбординге, чтобы запустить все, например на данных из http://ngodata.ru/dataset/kremlinlaws надо вручную скачать архив и положить в data/raw и структура датасета должна состоять из text/topics/date? Как в инструкции по запуску пайплайна?

**Comment by vvssttkk** [comment]
> @iggisv9t Я правильно понимаю, что по процедуре, которая описана в SoC онбординге, чтобы запустить все, например на данных из http://ngodata.ru/dataset/kremlinlaws надо вручную скачать архив и положить в data/raw и структура датасета должна состоять из text/topics/date? Как в инструкции по запуску пайплайна?

да. вот тут – https://github.com/newsviz/newsviz/wiki/Инструкция-по-запуску – чуть больше описано что к чему

**Comment by ezhdi** [comment]
Здесь http://ngodata.ru/dataset/govtwitters каждый твиттер-аккаунт лежит в отдельной папке (например, Aleksei_Kudrin) в .json файле. Для запуска нужно сливать все файлы в один?

**Comment by iggisv9t** [comment]
@ezhdi не обязательно один файл, можно собрать несколько если один выходит слишком большой, т.к. входные файлы по одному целиком загружаются в память. Актуальная инструкция по запуску для ветки stage лежит здесь https://github.com/newsviz/newsviz/blob/stage/docs/first_run.md

В любом случае все входные данные надо переделать в csv ожидаемого формата.
